althea_kernel_interface/src/hardware_info.rs

```diff
@@ -345,6 +349,70 @@ fn get_wifi_devices() -> Vec<WifiDevice> {
     ret
 }
 
+fn get_conntrack_info() -> Option<ConntrackInfo> {
+    let max_path = "/proc/sys/net/netfilter/nf_conntrack_max";
+    let table_count_path = "/proc/sys/net/netfilter/nf_conntrack_count";
+    let curr_conns_path = "/proc/net/nf_conntrack";
```

Comment:
Let me know if my understanding is correct. I writing this based on this article
https://morganwu277.github.io/2018/05/26/Solve-production-issue-of-nf-conntrack-table-full-dropping-packet/

nf_conntrack_max is total number of connections possible
nf_conntrack_count is number of table entries in a present moment, seems like OS changes this continuously
nf_conntrack shows how many connections are present at a given time in the kernel

Reply:
this looks correct, they should be constants though.